src/test/scala/org/broadinstitute/dsde/workbench/avram/db/DbSingleton.scala

```diff
@@ -0,0 +1,16 @@
+package org.broadinstitute.dsde.workbench.avram.db
+
+import com.typesafe.config.ConfigFactory
+import net.ceedubs.ficus.Ficus._
+import org.broadinstitute.dsde.workbench.avram.TestExecutionContext
+import org.broadinstitute.dsde.workbench.avram.config.DbcpDataSourceConfig
+
+
+// initialize database tables and connection pool only once
+object DbSingleton {
```

Comment:
I'm not sure why we need this. May be best to chat in person. My feeling is that, if we need this, we're doing something wrong in setting up our app/config.